From cdb5a99a79bd8001fd66f869cf94930aab5c2cd1 Mon Sep 17 00:00:00 2001
From: Quoc Ngo <quoc.ngo.wz@rvc.renesas.com>
Date: Mon, 6 Aug 2018 14:56:35 +0700
Subject: [PATCH] declarative-camera: correct framerate to fix "not negotiated"
 issue

this is a workaround to gstreamer's element can get correct caps

Signed-off-by: Quoc Ngo <quoc.ngo.wz@rvc.renesas.com>
---
 examples/multimedia/declarative-camera/declarative-camera.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/multimedia/declarative-camera/declarative-camera.qml b/examples/multimedia/declarative-camera/declarative-camera.qml
index 25c44e9..9f248fe 100644
--- a/examples/multimedia/declarative-camera/declarative-camera.qml
+++ b/examples/multimedia/declarative-camera/declarative-camera.qml
@@ -96,7 +96,7 @@ Rectangle {

         videoRecorder {
              resolution: "640x480"
-             frameRate: 30
+             frameRate: 30000/1001
         }
     }

--
1.9.1
